kit/src/main/java/com/oracle/javafx/scenebuilder/kit/util/eventnames/ImportBuilder.java

```diff
@@ -46,18 +79,15 @@ public static boolean isEmpty() {
     }
 
     /**
-     * Builds a new import statement. The key may be the "import" keyword, or the "import" keyword combined with a
+     * Builds the list of import statements. The key may be the "import" keyword, or the "import" keyword combined with a
      * "javafx.scene.input." package name. The value may be a full class name (e.g. with package) or an input event name.
      *
-     * @return new import statement
+     * @return list of new import statements
      */
-    public static String build() {
-        StringBuilder sb = new StringBuilder();
-        imports.forEach((key, value) -> {
-            sb.append(key);
-            sb.append(value);
-            sb.append(";\n");
-        });
-        return sb.toString();
+    public static List<String> build() {
+        return imports.entrySet()
+                      .stream()
+                      .map(preparedImport->preparedImport.getKey()+preparedImport.getValue())

```

Comment:
I didn't mean indentation (it was fine), but white/blank spaces in the lambda expression, like:
```
.map(preparedImport -> preparedImport.getKey() + preparedImport.getValue())
```